### PR TITLE
Outer Maridia: Use Flash Suit

### PR DESCRIPTION
**Pull request by kjbranch**

### CODE REVIEW

**Comment by osse101** [comment]
Its also unclear how `canSuitlessMaridia` fits.  If someone doesn't want to do any water rooms without gravity, there is a bit of platforming but not really much.  Climbing left of fish tank would take underwater platforming but that is an optimization of sparking from the ground, and falling down the right side wouldn't be water platforming.  And knowing how the horizontal component of shinesparks are affected by water doesn't seem important.  Jumping out before shinesparking, like with the everest hero shots would be `canSuitlessMaridia` I think.

File: region/maridia/outer/Mt. Everest.json [hunk 1]
```diff
@@ -1151,6 +1197,23 @@
         "FIXME: Check if this strat works through Toilet (either from PCJR or Crocomire's Room)"
       ]
     },
+    {
+      "link": [2, 6],
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/outer/Mt. Everest.json:1201]: Going to 6 can be done with more excess frames, so I put it there. But I could move it if you think it's better. I dont see any use to go to 8 except to then go to 6.

File: region/maridia/outer/Mt. Everest.json [hunk 2]
```diff
@@ -2157,6 +2295,15 @@
       },
       "requires": []
     },
+    {
+      "link": [4, 1],
+      "name": "Use Flash Suit",
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/outer/Mt. Everest.json:2300]: Good catch. Not seeing a way to do [7, 1] though.

**Comment by kjbranch** [comment]
I feel like `canSuitlessMaridia` should just be assumed at this difficulty - I dont imagine anyone turning on flash suits and turning it off. I wonder if something should be done about the tech in general though to make it more consistently used properly. Converting it to `canUnderwaterPlatform` and making a new variant `canTrickyUnderwaterPlatform` might help?

**Pull request merged**